src/vcSettingsUI.cpp

```diff
@@ -768,63 +724,107 @@ void vcSettingsUI_BasicMapSettings(vcState *pProgramState, bool alwaysShowOption
       }
     }
 
-    ImGui::TextUnformatted(vcString::Get("settingsMapType"));
+    ImGui::TextUnformatted(vcString::Get("settingsMapMapLayers"));
 
-    ImGui::BeginChild("mapSelection", ImVec2(512, 300));
+    pProgramState->settings.maptiles.activeLayerCount = 0;
+    for (int mapLayer = 0; mapLayer < vcMaxTileLayerCount; ++mapLayer)
+    {
+      if (ImGui::Button(udTempStr(pProgramState->settings.maptiles.layers[mapLayer].enabled ? "-##mapLayer%d" : "+##mapLayer%d", mapLayer), ImVec2(24, 24)))
+        pProgramState->settings.maptiles.layers[mapLayer].enabled = !pProgramState->settings.maptiles.layers[mapLayer].enabled;
 
-    ImGuiStyle &style = ImGui::GetStyle();
-    ImVec2 button_sz(128, 128);
-    float window_visible_x2 = ImGui::GetWindowPos().x + ImGui::GetWindowContentRegionMax().x;
+      ImGui::SameLine();
+      ImGui::Text("%s %d", vcString::Get("settingsMapLayer"), mapLayer);
```

Comment:
The number should be part of the string, I believe we've done this for other counts (search for `{0}` in the JSON file), also this should be `mapLayer + 1` otherwise it's `Layer 0` instead of `Layer 1`.

Reply:
Done and done.